client/src/raw.rs

```diff
@@ -1124,6 +1124,52 @@ proxier! {
             r.extensions_mut().insert(labels);
         }
     );
+    (
+        shutdown_worker,
+        ShutdownWorkerRequest,
+        ShutdownWorkerResponse,
+        // TODO: need the namespaced_request label?
+        |r| {
```

Comment:
@Sushisource I added these in by default to match the majority, but lmk if this should be removed

Reply:
Yeah it should be kept. All namespaced calls should have this label.

Reply:
oops, forgot to remove this comment, will remove in the follow-up PR